conjure-java-core/src/integrationInput/java/com/palantir/product/AsyncEteBinaryService.java

```diff
@@ -0,0 +1,21 @@
+package com.palantir.product;
+
+import com.google.common.util.concurrent.ListenableFuture;
+import com.palantir.dialogue.BinaryRequestBody;
+import com.palantir.tokens.auth.AuthHeader;
+import java.io.InputStream;
+import java.nio.ByteBuffer;
+import java.util.Optional;
+import javax.annotation.Generated;
+
+@Generated("com.palantir.conjure.java.services.dialogue.DialogueServiceGenerator")
+public interface AsyncEteBinaryService {
+    ListenableFuture<InputStream> postBinary(AuthHeader authHeader, BinaryRequestBody body);
+
+    ListenableFuture<Optional<ByteBuffer>> getOptionalBinaryPresent(AuthHeader authHeader);
```

Comment:
I know useImmutableBytes was an opt-in thing before, can we just start dialoge on com.palantir.conjure.java.lib.Bytes straight away?

Reply:
This should use a streamed binary type, not bytebuffer or bytes, so I’m not sure there’s a legitimate case for forcing useImmutableBytes here

Reply:
Ah yep good point